tests/Tests/Unit/OperationResponseTest.php

```diff
@@ -353,6 +358,26 @@ public function testPollingCastToInt()
         $this->assertEquals($op->getSleeps(), [3, 4, 6]);
     }
 
+    public function testLROOperationsClient()
+    {
+        $operationClient = $this->prophesize(LROOperationsClient::class);
+        $request = new GetOperationRequest(['name' => 'test-123']);
+        $operationClient->getOperation(Argument::exact($request))
+            ->shouldBeCalledOnce()
+            ->willReturn(new Operation());
+        $request = new DeleteOperationRequest(['name' => 'test-123']);
+        $operationClient->deleteOperation(Argument::exact($request))
+            ->shouldBeCalledOnce();
+        $request = new CancelOperationRequest(['name' => 'test-123']);
+        $operationClient->cancelOperation(Argument::exact($request))
+            ->shouldBeCalledOnce();
+
+        $operationResponse = new OperationResponse('test-123', $operationClient->reveal());
+        $operationResponse->reload();
+        $operationResponse->delete();
+        $operationResponse->cancel();
```

Comment:
nit: let's make these separate test cases

Reply:
Moved. thank you